airbyte-cdk/bulk/core/load/build.gradle

```diff
@@ -10,3 +23,18 @@ dependencies {
     testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.8.1")
     implementation "org.jetbrains.kotlin:kotlin-reflect:2.0.20"
 }
+
+task integrationTest(type: Test) {
+    description = 'Runs the integration tests.'
+    group = 'verification'
+    testClassesDirs = sourceSets.integrationTest.output.classesDirs
+    classpath = sourceSets.integrationTest.runtimeClasspath
+    useJUnitPlatform()
+    mustRunAfter tasks.check
+}
+configurations {
+    integrationTestImplementation.extendsFrom testImplementation
+    integrationTestRuntimeOnly.extendsFrom testRuntimeOnly
+}
+// These tests are lightweight enough to run on every PR.
+rootProject.check.dependsOn(integrationTest)
```

Comment:
it's weird that you set this `check -> integrationTest` dependency on the root project when the others all have `integrationTest -> check`. What's the intent here?

Reply:
Oh wait nevermind I misread, it's always `integrationTest -> check`. All good!